Formula/texlab.rb

```diff
@@ -30,7 +30,7 @@ def install
         {"jsonrpc": "2.0", "id": 0, "method": "initialize", "params": { "rootUri": null, "capabilities": {}}}
 
       EOF
-      assert_match "Content-Length: 543", stdout.gets("\n")
+      assert_match "Content-Length: 592", stdout.gets("\n")
```

Comment:
is there no other output to check?

Reply:
There is, but I was advised that a minimal test is sufficient. 

I can try to expand the test to check the full (multiline) output, but that's independent of the version bump.

Reply:
For the record, the full output is
```
Content-Length: 592

{"jsonrpc":"2.0","result":{"capabilities":{"completionProvider":{"resolveProvider":true,"triggerCharacters":["\\","{","}","@","/"," "]},"definitionProvider":true,"documentFormattingProvider":true,"documentHighlightProvider":true,"documentLinkProvider":{"resolveProvider":false},"documentSymbolProvider":true,"foldingRangeProvider":true,"hoverProvider":true,"referencesProvider":true,"renameProvider":{"prepareProvider":true},"textDocumentSync":{"change":1,"openClose":true,"save":{"includeText":false}},"workspaceSymbolProvider":true},"serverInfo":{"name":"TexLab","version":"2.0.0"}},"id":0
```

Reply:
while a minimal test is sufficient, the amount of bytes in the output seems to be variable between releases. Maybe you can just check for the "content-length" part? Or some other string in the output that will rarely change.

Reply:
That's the first time it's changed (and it's a major release); it's also possible (even likely) be that the underlying LSP protocol will be updated at some point, requiring further changes.

I don't mind (such minor) adaptations of the test as part of the version bump, but I'd be happy to change the test however you think best!

Reply:
Since there's no need to validate the output format beyond "no errors" I think leaving the amount of bytes out is good enough a test.

Reply:
Ok, I changed the test. Just to make sure: the assert should then be simply
```ruby
assert_match "Content-Length:"
```
right?